identity-and-access-management/secret-manager/concepts.mdx

```diff
@@ -88,12 +90,29 @@ This ensures the protection of critical information against accidental deletion
   Although a protected secret cannot be deleted, its versions can.
 </Message>
 
+## Secret types
+
+Secret types refer to the different kinds of sensitive data you can store with Secret Manager. Using secret types guarantees that all mandatory fields are present, facilitates the application of specific policies, and enhances integration with other products. The following secret types are available:
+
+- **Opaque**: this secret type is used for storing any kind of sensitive data. By default, secrets are opaque.
+
+- **Database credentials**: this secret type is used for storing usernames and passwords, or other authentication methods. Database credentials are used to access databases.
```

Comment:
This "other authenticatipn methods" shpuld be removed. We can say what are the fields I think:
- Username
- Password
- Engine
- Host
- Database name
- Port